pytrip/vdx.py

```diff
@@ -855,18 +882,15 @@ def calculate_center(self):
             self.center_pos = center_pos
             return center_pos
 
-    def get_color(self, i=None):
+    def get_color(self):
         """
-        :param int i: selects a colour, default if None.
         :returns: a [R,G,B] list.
         """
-        if i is None:
-            return self.color
-        return self.colors[i % len(self.colors)]
+        return self.color
```

Comment:
Do we really need a getter ?

Reply:
I think we do, its easier to mantain code in my opinion

Reply:
ok